src/browser/components/TabNavigation/Navigation.tsx

```diff
@@ -51,6 +50,7 @@ export interface NavItem {
   name: string
   title: string
   icon: (isOpen: boolean) => JSX.Element
+  // Need to clean up code and properly type content
```

Comment:
while I agree it's something we'd need to do, it feel unrelated to this change

Reply:
Yeah, we have the ideal of keeping PRs small and scoped to only the task at hand, but we also have a desire to gradually add types as we do other work.

Maybe we want to say that it's ok to add types, but only if it's in a file that is already being modified by the PR?

Reply:
I personally don't mind unrelated type touch-ups but I was mostly saying it's not _needed_  for this pr

Reply:
I notice I was not very clear in my comment

Reply:
@jharris4 @OskarDamkjaer - it's unrelated. I just added a comment so it reminds us to clean it up at some point. Also I noticed these components are a bit over complicated as many layers are nested. So, IMO probably better to touch it when we want to refactor our components.

Reply:
Generally we don't leave any todo/reminder comments in the code, we'd rather put a card in trello. While I prefer keeping the source clean from those kinds of comments, I'm open to discussing it w/ you and Eija if you guys think we should start doing so!